tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/ui/application/NewApplicationWizardModelTest.java

```diff
@@ -284,7 +288,22 @@ public void getParametersShouldReturnAParameterMapWhenTemplateIsNotNull() {
 		
 		assertArrayEquals(parameters.values().toArray(), model.getParameters().toArray());
 	}
-	
+
+	@Test
+	public void setWrongJsonAsTemplateFile() throws Exception {
+		IRoute route = Mockito.mock(IRoute.class);
+		when(route.getKind()).thenReturn(ResourceKind.ROUTE);
+		when(factory.create(any(InputStream.class))).thenReturn(route);
+		NotATemplateException ex = null;
+		try {
+			model.setLocalTemplateFileName("resources/jboss_infinispan-server_ImageStreamImport.json");
```

Comment:
why not simply assertFalse("No NotATemplateException occurred") here, leaving out the more complicated usage of the ex variable?